hack/kind-e2e.sh

```diff
@@ -2,66 +2,46 @@
 
 set -euo pipefail
 
-ROOT=$(cd $(dirname ${BASH_SOURCE[0]})/.. && pwd)
-cd ${ROOT}
+CLUSTER=${CLUSTER:-"e2e-mo-kind"}
+MO_VERSION=${MO_VERSION:-"nightly-20eeb7c9"}
+IMAGE_REPO=${IMAGE_REPO:-"matrixorigin/matrixone"}
```

Comment:
Don't repeat yourself, we should keep the default MO_VERSION and IMAGE_REPO in a single place